portability-core/src/main/java/org/dataportabilityproject/serviceProviders/google/contacts/GoogleContactsService.java

```diff
@@ -75,55 +87,37 @@ static VCard convertPersonToModel(Person person) throws IOException {
     Source: https://developers.google.com/people/api/rest/v1/people#personmetadata
     */
 
-    Pair<StructuredName, StructuredName[]> namesPair = convertToVCardNames(person.getNames());
-    vCard.setStructuredName(namesPair.first);
-    vCard.setStructuredNameAlt(namesPair.second);
-
-    /* TODO(olsona): uncomment when we want to test it
-    for (Telephone telephone : convertToVCardTelephoneNumbers(person.getPhoneNumbers())) {
-      vcard.addTelephoneNumber(telephone);
+    if (person.getNames() != null) {
+      Pair<StructuredName, StructuredName[]> namesPair = convertToVCardNames(person.getNames());
```

Comment:
Not a requirement but another way to do this without using a pair structure is to pass the vCard object and the person.getNames() to the method and have the method set the two fields.

Reply:
It'll be in the next PR.  Cleaner that way.